<ui version="4.0" >
 <class>BtBrowserWidget</class>
 <widget class="QWidget" name="BtBrowserWidget" >
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTreeWidget" name="lstBacktrace" >
     <property name="horizontalScrollBarPolicy" >
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="itemsExpandable" >
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus" >
      <bool>true</bool>
     </property>
     <property name="columnCount" >
      <number>4</number>
     </property>
     <column>
      <property name="text" >
       <string>#</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>File</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Line</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Function</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="lblStatus" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>31</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnConfigure" >
       <property name="text" >
        <string>Configure Paths...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClipboard" >
       <property name="text" >
        <string>Use Clipboard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBacktrace" >
       <property name="text" >
        <string>Load File...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
